src/parseCheckovConfig.ts

```diff
@@ -0,0 +1,19 @@
+import * as path from 'path';
+import { existsSync } from 'fs';
+import { getWorkspacePath } from './utils';
+import { Logger } from 'winston';
```

Comment:
Please move this import to up,
We're keeping an import order:
Vendor -> relative